distribution/openrct2.d.ts

```diff
@@ -211,6 +211,7 @@ declare global {
         subscribe(hook: "network.authenticate", callback: (e: NetworkAuthenticateEventArgs) => void): IDisposable;
         subscribe(hook: "network.join", callback: (e: NetworkEventArgs) => void): IDisposable;
         subscribe(hook: "network.leave", callback: (e: NetworkEventArgs) => void): IDisposable;
+        subscribe(hook: "rideRatings.calculate", callback: (e: RideRatingsCalculateArgs) => void): IDisposable;
```

Comment:
I would probably rename this to `ride.ratings.calculate` since as far as plugins are concerned, ratings are just part of ride and not a separate module.

Reply:
Done